facia/app/utils/DotcomFrontsLogger.scala

```diff
@@ -0,0 +1,48 @@
+package utils
+
+import common.GuLogging
+import common.LoggingField._
+import model.PressedPage
+import play.api.mvc.RequestHeader
+
+import scala.util.Random
+
+case class DotcomFrontsLogger() extends GuLogging {
+  def logFieldFromRequest()(implicit request: RequestHeader): List[LogField] = {
+    List[LogField](
+      "req.method" -> request.method,
+      "req.url" -> request.uri,
+      "req.id" -> Random.nextInt(Integer.MAX_VALUE).toString,
+    )
+  }
+
+  def logFieldFromProperties(properties: Map[String, String]): List[LogField] =
+    properties.map({ case (k, v) => LogFieldString(k, v) }).toList
```

Comment:
I love `k`'s and `v`'s

Reply:
I'm going to blame wherever I stole this code from for that! Thanks!